lib/features/steps/error_reporting_steps.rb

```diff
@@ -169,6 +182,19 @@
   step "the error payload field \"events.0.#{field}\" matches the JSON fixture in \"#{fixture_path}\""
 end
 
+Then('the event {string} string is empty') do |keypath|
```

Comment:
Is this supposed to be `{string} string` or just `{string}`?

Reply:
I think the wording could possibly be improved, but it is deliberate and what Android has always used (this change absorbs the existing step into MazeRunner).